test/browser/whiteboard.js

```diff
@@ -171,16 +171,6 @@ describe('Whiteboard tests', async function() {
         assert(!whiteboard.elements);
     });
 
-    it('function: undoWhiteboard, raises event: whiteboardSync', async () => {
```

Comment:
Are you removing this?

Reply:
Yes, it was removed on beta because it was a wrong test.